src/tools/pub/cmd/index.md

```diff
@@ -1,132 +1,152 @@
 ---
 title: The pub tool
-description: Pub, a package management tool for Dart, supports a variety of commands.
+description: The command-line interface for pub, a package management tool for Dart.
 ---
 
-The `pub` tool has commands for managing packages
-and for deploying packages and command-line apps.
-For general information about using the pub package manager, see
-[How to use packages](/guides/packages).
+The [pub package manager](/guides/packages) has a command-line interface
+that works with either the
+[`flutter` tool][flutter-cli] or the [`dart` tool][dart-cli].
+With either tool, add the `pub` command followed by
+a subcommand such as `get`:
 
-{% include flutter-packages.md %}
+```terminal
```

Comment:
Swap these two as we're on the Dart site.

Reply:
I had two reasons for ordering it this way:
* I wanted to make sure people saw that they might want to run it using the `flutter` command. Everywhere else on this site, we show only the `dart` command.
* It made it easy to just say "any other package" for the `dart` comment.

Reply:
I really, really don't like listing Flutter first on the Dart.dev site. It just seems wrong.

I get the "any other part", but I also don't think that level of detail is needed in the comments (they could just say "Get dependencies for a Dart package"/"Get dependencies for a Flutter package")

Reply:
That seems wrong to me because Flutter packages _are_ Dart packages. I'd say, instead "Get dependencies for a non-Flutter package".

I guess what it comes down to is: Is there harm in using `dart` when you should use `flutter`? Or vice versa? Which mistake is most likely to occur and cause problems?

If using `dart` when you need `flutter` isn't likely to cause confusion, then I'm OK with putting `dart` first.

Reply:
There are some small differences, but those are being removed.

For what it's worth, I've heard lots of folks call "Dart packages that contain Flutter specific code" for simply "Flutter packages" and never experienced much confusion over that.

Reply:
I agree that "Flutter packages" seems fairly clear. It's "Dart packages" that is ambiguous, because it's unclear whether it includes Flutter packages.

Reply:
I get the general point, but not sure I find "Dart packages" ambiguous when it's listed right next to "Flutter packages". I don't care that strongly though.

Reply:
I've swapped them and tweaked the comments.